docs/architecture/modernize-desktop/index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Modernizing Desktop Apps on Windows 10 with .NET Core 3.1
-description: Learn how to modernize existing desktop apps with .NET Core 3.1
-ms.date: 05/12/2020
+title: Modernizing Desktop Apps on Windows 10 with .NET
+description: Learn how to modernize existing desktop apps with .NET
+ms.date: 12/29/2020
 ---
-# Modernizing Desktop Apps on Windows 10 with .NET Core 3.1
+# Modernizing Desktop Apps on Windows 10 with .NET
```

Comment:
We should name it `Modernizing Desktop Apps on Windows 10 with .NET 5` and not just just .NET

Reply:
Ok, sure. Will update the title.

Reply:
Updated.